diff --git a/src/Assets.js b/src/Assets.js
index 0870a4c62c7ef9e7fb3a93c0bed74aa8ea5a765e..ccd48306524b45ac8586364c053292a835bbe9a7 100644
--- a/src/Assets.js
+++ b/src/Assets.js
@@ -167,9 +167,10 @@ async function getAssetData(
   assetUrlPath = (0, _pathUtils.normalizePathSeparatorsToPosix)(assetUrlPath);
   const isImage = isAssetTypeAnImage(_path.default.extname(assetPath).slice(1));
   const assetInfo = await getAbsoluteAssetInfo(assetPath, platform);
-  const isImageInput = assetInfo.files[0].includes(".zip/")
+  // image-size 2 only accepts bytes, not filenames.
+  const isImageInput = isImage
     ? _fs.default.readFileSync(assetInfo.files[0])
-    : assetInfo.files[0];
+    : null;
   const dimensions = isImage ? (0, _imageSize.default)(isImageInput) : null;
   const scale = assetInfo.scales[0];
   const assetData = {
